.github/workflows/sync-testnets.yml

```diff
@@ -0,0 +1,150 @@
+name: Sync Testnets
+
+on:
+  push:
+    branches: ["master"]
+
+jobs:
+  create_docker_image:
+    runs-on: ubuntu-latest
+    outputs:
+      docker_tag: ${{ steps.prepare_ref.outputs.docker_tag }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.ref }}
+        
+    - name: Prepare docker tag
+      id: prepare_ref
+      run: |
+        REF_NAME=${{ github.event.pull_request.head.ref }}
+        CLEAN_REF=$(echo "${REF_NAME/refs\/heads\//}" | sed 's/[^a-zA-Z0-9._-]/-/g')
+        echo "CLEAN_REF=$CLEAN_REF" >> $GITHUB_ENV
+        echo "::set-output name=docker_tag::$CLEAN_REF"
```

Comment:
This would generate a warning as it's deprecated. Replace with
```sh
echo "docker_tag=$CLEAN_REF" >> $GITHUB_OUTPUT
```

Reply:
Done, thanks! Also updated one task to latest version